test/unit/io/sam_file/format_sam_test.cpp

```diff
@@ -21,13 +21,13 @@ struct sam_file_read<seqan3::format_sam> : public sam_file_data
 	using stream_type = std::istringstream;
 
     std::string big_header_input{
-R"(@HD	VN:1.6	SO:coordinate	SS:coordinate:queryname	GO:none
-@PG	ID:qc	PN:quality_control	CL:qc -f file1	DS:trim reads with low qual	VN:1.0.0
-@PG	ID:novoalign	PN:novoalign	VN:V3.02.07	CL:novoalign -d /path/hs37d5.ndx -f /path/file.fastq.gz	PP:qc
+R"(@HD	VN:1.6	SO:coordinate	SS:coordinate:queryname	GO:none	pb:usr_defined_tag
+@PG	ID:qc	PN:quality_control	CL:qc -f file1	DS:trim reads with low qual	VN:1.0.0	pb:usr_defined_tag
+@PG	PN:novoalign	ID:novoalign	VN:V3.02.07	CL:novoalign -d /path/hs37d5.ndx -f /path/file.fastq.gz	PP:qc
 @SQ	SN:ref	LN:249250621
-@SQ	SN:ref2	LN:243199373	AS:hs37d5
+@SQ	SN:ref2	LN:243199373	AS:hs37d5	pb:usr_defined_tag
 @RG	ID:U0a_A2_L1	PL:illumina	PU:1	LB:1	SM:NA12878
-@RG	ID:U0a_A2_L2	PL:illumina	SM:NA12878	PU:1	LB:1
+@RG	ID:U0a_A2_L2	PL:illumina	SM:NA12878	PU:1	LB:1	pb:usr_defined_tag
 @CO	Tralalalalalala this is a comment
 )"};
```

Comment:
Please add new test cases for the individual changes and leave the old ones as they are.
I.e. a two line header for swapping `PN` and `ID` and a two line header for adding a user defined tag